synapse/cortex.py

```diff
@@ -1349,6 +1357,29 @@ async def initServicePassive(self):
         for layer in self.layers.values():
             await layer.initLayerPassive()
 
+    async def _runHotfixes(self):
+        if not self.conf.reqConfValu('storm:hotfixes'):
+            return
+
+        # Run the hotfixes in their own asyncio task.
+        async def hotfixes():
+            text = '$lib.cell.hotFixesApply()'
+            async for item in self.storm(text):
+                if item[0] in ('print', ):
+                    logger.debug(f'Hotfix message: {item[1].get("mesg")}')
+                    continue
+                if item[0] in ('warn', ):
+                    logger.warning(f'Hotfix message: {item[1].get("mesg")}')
+                    continue
+                if item[0] in ('err', ):
+                    logger.error(f'Hotfix error: {item}')
+
+        fut = self.schedCoro(hotfixes())
```

Comment:
Why are we running this in a schedCoro() ( and then doing crazy racy sleep loop stuff? ) :)